src/Dapper.AOT.Analyzers/Internal/Inspection.cs

```diff
@@ -2,6 +2,8 @@
 using Dapper.Internal.Roslyn;
 using Dapper.SqlAnalysis;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
```

Comment:
see https://github.com/DapperLib/DapperAOT/tree/main/src/Dapper.AOT.Analyzers/Internal/Roslyn - I've moved all the language-specific stuff *out* - the analyzers now support both VB and C#, which needed an extra layer of abstraction. No code outside of `LanguageHelpers` should ref `.CSharp`

Reply:
refactored to not have language-specific stuff in the `Inspection`